app/pages/project/home/project-home.spec.js

```diff
@@ -68,6 +66,34 @@ describe('ProjectHome', function() {
     expect(wrapper.find('ProjectMetadata')).to.have.lengthOf(1);
   });
 
+  describe('with a launch-approved project', function () {
+    project.launch_approved = true;
+    wrapper = shallow(
+      <ProjectHome
+        project={project}
+        background={background}
+        translation={translation}
+      />);
+    const disclaimer = wrapper.find('Translate[className="project-disclaimer"]');
```

Comment:
I'd like to move away from relying on class names as selectors as this can be brittle and if we do indeed go forward with styled-components, then we won't be able to use class names since they're generated. Enzyme supports object properties as selectors too:
`wrapper.find({ content: 'project.disclaimer' })`

Reply:
Nice. I agree moving away is a good idea, regardless of stlyed-components, which I still find pretty awkward. What made you think of trying them out, i.e. advantages?

Reply:
A few reasons:
- It'll be better and easier to share styles in shared components using JS in CSS. It would make sure the styles are correctly applied and have no issues with trying to setup to ingest CSS from a package, setup webpack, class name collisions, etc. 
- It has a nice theming feature which will make it easier to manage which theme is used with app state. The classifier redesign will have a light and dark theme.
- Grommet v2 uses styled-components for ease of theming. 
- We're interested in using Grommet with the classifier redesign.

Reply:
Cool, seems like a good idea then, thanks.